spec/factories/user_factory.rb

```diff
@@ -10,6 +10,11 @@
       admin true
     end
 
+    trait :supervisor do
+      admin      true
```

Comment:
Admin and supervisor at the same time?
It is more like a role and we should extract it to a separate table.
Do you agree? @sveredyuk

Reply:
Maybe policies?

Reply:
I agree
But we out of time. We need quick solution ASAP.

Reply:
@denysmedynskyi Sorry, but I need merge this to protect sensitive information from uncontrolled access by courses admins,

Reply:
And with new students we can develop role system and drop with quick workaround.

Reply:
I understand ASAP-ness.